src/main/java/org/generationcp/middleware/service/api/dataset/ObservationUnitsSearchDTO.java

```diff
@@ -19,7 +19,7 @@ public class ObservationUnitsSearchDTO extends SearchRequestDto {
 	private List<MeasurementVariableDto> environmentConditions;
 	private List<String> genericGermplasmDescriptors;
 	private List<String> additionalDesignFactors;
-	private List<MeasurementVariableDto> selectionMethodsAndTraits;
+	private List<MeasurementVariableDto> selectionMethodsAndTraitsAndAnalysisSummary;
```

Comment:
@mderamos I suggest to use datasetVariables instead and add a comment with the variable types that the list contains based on the dataset type.